op-guide/configuration.md

```diff
@@ -50,3 +50,152 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
+
+### name
+
++ The human-readable unique name for this PD member.
++ default: "pd"
++ If you want to start multiply PDs, you must use different name for each one.
+
+### --host
+
++ The server host IP. 
++ default: "127.0.0.1"
++ You must set this flag so that it can be connected from outside., and can only ignore this in standalone test. 
+
+### --data-dir
+
++ The path to the data directory.
++ default: "default.{name}"
+
+### --client-port
+
++ The listening port for client traffic. 
++ default: 2379
+
+### --advertise-client-port
+
++ The advertise port for client traffic.
++ default: {client-port}
++ If you use Docker to start PD, and use different ports to map the listening client port, you must set this flag. For example, if you use `-p 12379:2379`, the advertise client port must be 12379 through which the client can connect to PD.
+
+### --peer-port
+
++ The listening port for peer traffic.
++ default 2380
+
+### --advertise-peer-port
+
++ The advertise port for peer traffic.
++ default: {peer-port}
++ If you use Docker to start PD, and use different ports to map the listening peer port, you must set this flag. For example, if you use `-p 12380:2380`, the advertise peer port must be 12380 for other peers connecting.
+
+### --initial-cluster
+
++ The initial cluster configuration for bootstrapping. 
++ default: "{name}=http://{host}:{advertise-peer-port}"
++ You must set this flag to bootstrap PD cluster. For example, if you have three PDs (pd1, pd2, pd3), the initial cluster is "pd1=http://pd1_host:2380,pd2=http://pd2_host:2380,pd3=http://pd3_host:2380".
++ Note: this flag will be deprecated later.
+
+### --initial-cluster-state
+
++ The initial cluster state ("new" or "existing").
++ default: "new"
++ Note: this flag will be deprecated later.
+
+### --port
+
++ The listening port for client traffic.
++ default: 1234
++ Note: this flag will be deprecated later.
+
+### --advertise-port
+
++ The server advertise port for client traffic. 
++ default: {port}
++ If you use Docker to start PD, and use different ports to map the listening port, you must set this flag. For example, if you use `-p 11234:1234`, the advertise port must be 11234 through which the client can connect to PD.
++ Note: this flag will be deprecated later.
+
+### --http-port
+
++ The server http port.
++ default: 9090
++ Note: this flag will be deprecated later.
+
+## TiKV
+
+### -P, --port
+
++ The server listening port for outer traffic.
++ default: 20160
+
+### --advertise-port
+
++ The server advertise port for outer traffic. 
++ default: {port}
++ If you use Docker to start TiKV, and use different ports to map the listening port, you must set this flag. For example, if you use `-p 20161:20160`, the advertise port must be 20161 through which the client can connect to TiKV.
+
+### -H, --host
+
++ The server host IP.
++ default: "127.0.0.1"
++ You must set this flag so that it can be connected from outside., and can only ignore this in standalone test. 
```

Comment:
You must set this flag so that it can be connected from outside. You can only ignore it in the standalone mode.